src/arena.c

```diff
@@ -1886,6 +1886,33 @@ arena_muzzy_decay_ms_default_set(ssize_t decay_ms) {
 	return false;
 }
 
+bool
+arena_retain_grow_limit_get_set(tsd_t *tsd, arena_t *arena, size_t *old_limit,
+    size_t *new_limit) {
+	assert(opt_retain);
+
+	pszind_t new_ind JEMALLOC_CC_SILENCE_INIT(0);
+	if (new_limit != NULL) {
```

Comment:
It seems like this should either be an error, or, if we want a null new-limit to mean "unlimited", we should document that.

Reply:
Oh wait I'm dumb, there should be a way to get the current limit. nvm.

Reply:
Yep. NULL new_limit means reading the value.